control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -195,10 +233,17 @@ func Setup(ctx context.Context, opts *operator.HostedClusterConfigOperatorConfig
 		versions:                  opts.Versions,
 		operateOnReleaseImage:     opts.OperateOnReleaseImage,
 		ImageMetaDataProvider:     opts.ImageMetaDataProvider,
+		kubeSystemSecretInformer:  kubeSystemSecretInformer,
```

Comment:
why is this informer in the reconciler? seems unused

Reply:
nice catch. Thanks